package.json

```
@@ -12,12 +12,15 @@
   "devDependencies": {
     "eslint": "6.1.0",
     "eslint-config-htmlacademy": "0.5.2",
+    "moment-locales-webpack-plugin": "^1.2.0",
     "webpack": "4.44.1",
     "webpack-cli": "3.3.12",
     "webpack-dev-server": "3.11.0"
   },
   "engines": {
     "node": "10"
   },
-  "dependencies": {}
+  "dependencies": {
+    "moment": "^2.27.0"
```

версия должна быть зафиксирована
```suggestion
    "moment": "2.27.0"
```